src/App.jsx

```diff
@@ -14,70 +17,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const goodsList = goods.map((name, index) => ({
+  name,
+  key: index + 1,
+}));
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+export const App = () => {
+  const [goodSelected, setGoodSelected] = useState(goodsList[8]);
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {goodSelected
+          ? `${goodSelected.name} is selected`
+          : 'No goods selected'
+        }
+        {goodSelected && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setGoodSelected('')}
+          />
+        )}
+      </h1>
+      <table className="table">
+        <tbody>
+          {goodsList.map(good => (
+            <tr
+              key={good.key}
+              data-cy="Good"
+              className={good.key === goodSelected.key
+                && ('has-background-success-light')}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  key={good.key}
+                  onClick={() => {
+                    if (good.key === goodSelected.key) {
+                      setGoodSelected('');
+                    }
```

Comment:
Can we simplify it like `setGoodSelected(good.key === goodSelected.key ? ... : ...)`?